moduleframework/dockerlinter.py

```diff
@@ -184,15 +184,18 @@ def check_from_directive_is_valid(self):
 
     def check_chained_run_dnf_commands(self):
         """
-        Function checks if Dockerfile does not contain more `RUN dnf/yum` commands
-        in more then one row.
+        Function checks if Dockerfile does not contain more `RUN dnf/yum` commands in more then one row.
```

Comment:
The docstring might make more sense as `This function checks that there are no consecutive RUN commands executing dnf/yum in the Dockerfile, as these need to be chained.`